test/extended/olm/olmv1.go

```diff
@@ -109,7 +109,7 @@ var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLM
 
 		const (
 			packageName = "cluster-logging"
-			version     = "6.2.2"
+			version     = "6.3.0"
```

Comment:
Up to 3 hours ago, we did not have this content for index 4.20.
Do we have it now?